device.py

```diff
@@ -1,4 +1,6 @@
 from .baseapi import BaseAPI
+from resources import Message
+import sys
```

Comment:
I personally find sorting my imports sensibly helps reviewing my code.

I follow this rules:

- First python imports
- then third party libraries
- then project/package imports
- then local imports

All "sections" are kept sorted by name and separated by a blank line. I also put relative imports below absolute ones.